packages/terra-form-select/src/combobox/Menu.jsx

```diff
@@ -275,26 +274,6 @@ class Menu extends React.Component {
     }
   }
 
-  /**
-   * Sets the hovered option as the active value.
-   * @param {event} event - The mouse enter event.
-   * @param {ReactNode} option - The option that received the mouse enter event.
-   */
-  handleMouseEnter(event, option) {
-    // Prevents setting the active option on mouse enter if the keyboard scrolled the view.
-    if (this.scrollTimeout) {
-      return;
-    }
-
-    if (!option.props.disabled) {
-      this.setState({ active: option.props.value });
-    }
-
-    if (option.props.onMouseEnter) {
-      option.props.onMouseEnter(event);
-    }
-  }
-
```

Comment:
I understand that this is now being handled by CSS but will this cause any loss of functionality?